ballot/app/scripts/deploy_service.js

```diff
@@ -1,7 +1,7 @@
 const chalk = require('chalk');
 const oasis = require('@oasislabs/client');
 
-oasis.workspace.Ballot.deploy({
+oasis.workspace.Ballot.deploy('pikachu',  ['snorlex', 'ghastly'], {
```

Comment:
the first argument should be a description, the list of strings is the list of candidates

Reply:
I thought it was asking which was your favorite pikachu. Isn't the Ash guy a pikachu trainer?

Reply:
also ... what's a `snorlex` and `ghastly`?

Reply:
it should actually be snorlax. `ghastly` and `snorlax` are pokemons. These are the candidates we use in the secret ballot app

Reply:
"snorlax" and "gastly"

Reply:
you are correct!

Reply:
updated description to conform to this pattern!

Reply:
ty. also change the names of the pokemon, per the discussion above.